src/component/personinfo/components/etiketter.tsx

```diff
@@ -101,11 +101,21 @@ function Etiketter() {
             >
                 Reservert KRR
             </Fokus>
+            <Fokus
+                visible={
+                    oppfolging?.registrertKRR &&
+                    !oppfolging?.kanVarsles &&
+                    !oppfolging.manuell &&
+                    !oppfolging?.reservasjonKRR
+                }
+            >
+                Utdatert kontaktinformasjon i KRR
+            </Fokus>
             <Fokus visible={oppfolging?.inaktivIArena}>Inaktivert</Fokus>
             <Fokus visible={!oppfolging?.underOppfolging}>Ikke under oppfølging</Fokus>
             <Fokus visible={gjeldendeEskaleringsvarsel}>Varsel</Fokus>
             <Fokus
-                visible={!oppfolging?.reservasjonKRR && !oppfolging?.manuell && !oppfolging?.kanVarsles}
+                visible={!oppfolging?.registrertKRR}
```

Comment:
```suggestion
                visible={oppfolging?.registrertKRR === false}
```
Da vil den ikke vise taggen når den laster? (undefined)